tests/integration_tests/testsuite/ovc_tests/a_basic/cloudspaces_tests.py

```diff
@@ -42,29 +43,29 @@ def test001_create_cloudspace_with_wrong_params(self):
         self.log('%s STARTED' % self._testID)
 
         self.log('Create an cloudspace with providing non existing parameter, should fail')
-        self.cloudspaces[self.cs1] =  {'account': self.acc1, self.random_string(): self.random_string()}
+        self.cloudspaces[self.cs1] = {'account': self.acc1, self.random_string(): self.random_string()}
         res = self.create_cs(openvcloud=self.openvcloud, accounts=self.accounts, vdcusers=self.vdcusers,
                              cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
-        self.assertEqual(res,'parameter provided is wrong' )
+        self.assertEqual(res, 'parameter provided is wrong')
 
         self.log('Create an cloudspace with non-exist account , should fail')
-        self.cloudspaces[self.cs1] =  {'account': self.random_string}
+        self.cloudspaces[self.cs1] = {'account': self.random_string}
         res = self.create_cs(openvcloud=self.openvcloud, accounts=self.accounts, vdcusers=self.vdcusers,
                              cloudspaces=self.cloudspaces, temp_actions=self.temp_actions)
-        self.assertEqual(res,"account doesn't exist" )
+        self.assertEqual(res, "account doesn't exist")
 
         self.log('Create an cloudspace with providing non existing user, should fail')
         self.cloudspaces[self.cs1] = {'account': self.acc1,
                                       'users': OrderedDict([('name', self.random_string()),
-                                                          ('accesstype', 'CXDRAU')])}
+                                                            ('accesstype', 'CXDRAU')])}
```

Comment:
Adding users here is not applicable anymore.. please make sure it is covered in the action section